core/modules/filters/contains.js

```diff
@@ -0,0 +1,77 @@
+/*\
+title: $:/core/modules/filters/contains.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for comparing the contents of an input tiddlers field and the
+operand.
+
+This filter returns the input title iff the operand is a subset of the given
+field of the input.
+
+Example 1:
+
+MyList: 1 2
+filter: [[tiddler1]][[tiddler2]]+[contains<MyList>]
+
+tiddler1 list field: 1 2 foo hello
+
+tiddler1 is returned because the list field contains both 1 and 2
+
+tiddler2 list field: 1 foo hello
+
+tiddler2 is not returned because the list field doesn't contain 2
+
+Example 2:
+
+MyList: foo bar
+filter: [[tiddler1]][[tiddler2]]+[contains:tags<MyList>]
```

Comment:
ok. ... It works with every field. right?

Reply:
yes, it tries to parse it as a list so that may give strange behaviour, but that is the same as many of the list opts operators.